multinet/api/tasks/upload/csv.py

```diff
@@ -38,14 +38,18 @@ def process_row(row: Dict[str, Any], cols: Dict[str, TableTypeAnnotation.Type])
 
 @shared_task(base=ProcessUploadTask)
 def process_csv(
-    task_id: int, table_name: str, edge: bool, columns: Dict[str, TableTypeAnnotation.Type]
+    task_id: int, table_name: str, edge: bool, columns: Dict[str, TableTypeAnnotation.Type], delimiter: str, quotechar: str
 ) -> None:
     upload: Upload = Upload.objects.get(id=task_id)
 
     # Download data from S3/MinIO
     with upload.blob as blob_file:
         blob_file: BinaryIO = blob_file
-        csv_rows = list(csv.DictReader(StringIO(blob_file.read().decode('utf-8'))))
+        csv_rows = list(csv.DictReader(
+            StringIO(blob_file.read().decode('utf-8')),
```

Comment:
I realize you didn't write this, but just out of curiosity, what happens if we omit the `StringIO` intermediary buffer? Can `DictReader` work directly on `blob_file`? i.e.

```python
csv_rows = list(csv.DictReader(blob_file))
```

Reply:
I experimented with this on a new branch in #146. It seems that the StringIO made my upload more consistent. Without it, sometimes the iterator would be over letters instead of lines when processing the csv